htsohm/material_files.py

```diff
@@ -197,131 +179,95 @@ def write_child_definition_files(run_id, parent_id, generation, mutation_strengt
-        p_mix, unpack=True, skip_header=7, skip_footer=11, usecols=0, dtype=str
-    )
-    print('Mixing rules\nchem\tep\t\tsig')
-    for i in range(len(chemical_ids)):
-        print('%s\t%s\t%s' % (chemical_ids[i], old_epsilons[i], old_sigmas[i]))
-    atom_types = []
-    for i in range(len(chemical_ids)):
-        epsilon = round( old_epsilons[i] + mutation_strength * (uniform(*epsilon_limits) -
-        old_epsilons[i]), 4)
-        sigma = round(
-            old_sigmas[i] + mutation_strength * (uniform(*sigma_limits) - old_sigmas[i]
-        ), 4)
-        atom_type = {
-            "chemical-id" : chemical_ids[i],
-            "charge"      : 0.,    # See Docstring Todo.
-            "epsilon"     : epsilon,
-            "sigma"       : sigma}
-        atom_types.append(atom_type)
-
-        mix_file = os.path.join(child_forcefield_directory, 'force_field_mixing_rules.def')
-        write_mixing_rules(mix_file, atom_types)
-        print('  - force_field_mixing_rules.def\tWRITTEN.')
+    # perturb LJ-parameters
+    child_material.atom_types = []
+    for atom_type in parent_material.atom_types:    
+        new_atom_type = {'chemical-id' : atom_type['chemical-id']}
+        new_atom_type['epsilon'] = (round(atom_type['epsilon'] +
+                mutation_strength * (uniform(*epsilon_limits) -
+                atom_type['epsilon']), 4))
+        new_atom_type['sigma'] = (round(atom_type['sigma'] +
+                mutation_strength * (uniform(*epsilon_limits) -
+                atom_type['epsilon']), 4))
+        child_material.atom_types.append(new_atom_type)
 
     ########################################################################
-    # load values from parent cif-file
-    p_cif = os.path.join(mat_dir, run_id + '-' + str(parent.uuid) + '.cif')
-    n1, n2, old_x, old_y, old_z = np.genfromtxt(p_cif, unpack=True, skip_header=16)
-    old_atom_types = np.genfromtxt(p_cif, usecols=0, dtype=str, skip_header=16)
-    old_abc = np.genfromtxt(
-        p_cif, unpack=True, skip_header=4, skip_footer=len(old_x) + 9, usecols = 1
-    )
-    ########################################################################
     # calculate new lattice constants
-    old_a = old_abc[0]
-    old_b = old_abc[1]
-    old_c = old_abc[2]
-    random_a = round(uniform(*lattice_limits), 4)
-    random_b = round(uniform(*lattice_limits), 4)
-    random_c = round(uniform(*lattice_limits), 4)
-    a = round(old_a + mutation_strength * (random_a - old_a), 4)
-    b = round(old_b + mutation_strength * (random_b - old_b), 4)
-    c = round(old_c + mutation_strength * (random_c - old_c), 4)
-    lattice_constants = {"a" : a, "b" : b, "c" : c}
+    child_material.lattice_constants = {}
+    for i in ['a', 'b', 'c']:
+        old_x = parent_material.lattice_constants[i]
+        random_x = uniform(*lattice_limits)
+        new_x = round(old_x + mutation_strength * (random_x - old_x), 4)
+        child_material.lattice_constants[i] = new_x
+
```

Comment:
if you used a copy / perturb pattern, this could look something like: 

child_material.lattice_constants = parent_material.lattice_constants.copy()
for i in ['a', 'b', 'c']:
    old_x = parent_material.lattice_constants[i]
    random_x = uniform(*lattice_limits)
    child_material.lattice_constants[i] += mutation_strength * (random_x - old_x)

Probably a way of simplifying it more